src/plugins/menu/ajax/menu-include-en.html

```diff
@@ -70,6 +70,7 @@ <h2>Topics menu</h2>
 					</li>
 				</ul>
 			</li>
+			<li><a href="#" class="item">No Submenu</a></li>
```

Comment:
Can you replicate this new menu item in the french ajax file (menu-include-fr.html)

Reply:
I have made the text "Sans Sous-Menu" on the french page. If this is not appropriate, please change.